bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2633,11 +2696,11 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( name.substr(0,8) == "TVectorT" ) {  // allow proper iteration
+   else if (name.find("TVectorT") != std::string::npos) {
```

Comment:
`substr` would be faster than `find` here and below, as we know the position we expect `TVectorT` to be at

Reply:
I'll revert this change :+1: